drivers/util.py

```diff
@@ -1236,6 +1236,7 @@ def daemon():
 
 if __debug__:
     try:
+        #pylint: disable=used-before-assignment
```

Comment:
Actually, this entire block, from the `if __debug__` can be deleted, it's legacy cruft.